lib/tasks/handover_chase_email.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'rake'
+
+namespace :handover_chase_emails do
+  desc 'Send follow-up emails LDUs two weeks before handover if COM still not assigned'
+  task process: :environment do
+    HandoverFollowUpJob.perform_later(Time.zone.today + 14.days)
```

Comment:
I think there is a problem here (or a problem with my understanding). We are supposed to nudge when the handover date is 1 week past, however this date has 1 week substracted in the code (so it is today + 1.week in production) so I think this alerts 1 week *before* the handover is due rather than 1 week after. I think this should just be Time.zone.today?

Reply:
That should have been changed, thanks for spotting that